content/en/security_platform/cloud_workload_security/workload_security_rules.md

```diff
@@ -53,11 +61,11 @@ To ensure that your policy is evaluated in-kernel for maximum efficiency, always
 - `[event_type].file.name`
 - `[event_type].file.path`
 
-`[event_type]` can be open or exec, for example.
+**Note**: `[event_type]` can be open or `exec`, for example.
 
 Use wildcards (`*`) carefully. For example, never use `open.file.path =~ "*/myfile"`. If you must use wildcards prefixing directories, at least two levels are required: `open.file.path =~ "*/mydir/myfile")`.
```

Comment:
Can we add an explanation on why we don't recommend using wildcards - ie. can cause name conflict and ambiguities